environment/environment.go

```diff
@@ -111,25 +112,43 @@ func DeployEnvironment(config *Config) (*Environment, error) {
 	return e, e.SyncConfig()
 }
 
+// CommonRemoteRunnerValues builds the map with the common expected values for remote runner
+func CommonRemoteRunnerValues(testTag, slackAPI, slackChannel, slackUser string) map[string]interface{} {
+	return map[string]interface{}{
+		"remote_test_runner": map[string]interface{}{
+			"test_name":     testTag,
+			"slack_api":     slackAPI,
+			"slack_channel": slackChannel,
+			"slack_user_id": slackUser,
+		},
+	}
+}
+
 // DeployRemoteRunnerEnvironment is used for deploying environment with a remote runner inside a pod
 // suitable for a long-running tests
 func DeployRemoteRunnerEnvironment(
 	config *Config,
-	testName,
-	slackAPI,
-	slackChannel,
-	slackUser,
 	frameworkConfigPath,
 	networksConfigPath,
 	testExecutablePath string,
-	customEnvVars []string, // list of environment variables to pull from your env to use in the test
+	runnerHelmValues map[string]interface{},
```

Comment:
Why not map[string]string? Does it need to be able to accept ints and whatnot?

Reply:
It needs to accept maps for secrets and custom environment variables.